### PR TITLE
Upload only 3lap 1

### PR DESCRIPTION
**Pull request by salvoch**
Changed mk64.py to upload only the fastest 3-lap record instead of any top 5 record.

### CODE REVIEW

**Comment by gabranches** [comment]
LGTM 👍

File: src/utils/mk64.py [hunk 1]
```diff
@@ -5,20 +5,15 @@
...
-                # track_name = old_times['name']
-                # racer_name = racer_ids[nr_re['character']]
-                new_time = nr_re['time']
-                uploader.send_to_gus(new_time, TRACK_SLUGS[j], rtype="3lap")
-                print('New top 5 race time!')
-                break
+
+        three_lap_record = new_times['track-record'].records[0]
+        if old_times['track-record'].records[0] != three_lap_record and three_lap_record != None:
+            # Commented these out because not being used atm
```
Comment (jmct) [inline src/utils/mk64.py]: Why'd you keep these comments (that represent code we don't need), but delete the explanatory comments?

**Review by jmct — CHANGES_REQUESTED** [review]
Code is simpler, which is always nice!

Just add back the explanation of why we only check whether `nr_re` is `None`. It might be obvious to us now, but it took a bit of thinking on the day and future us won't remember.

**Review by jmct — APPROVED** [review]
Awesome, thanks!

**Pull request merged**